.github/actions/zango-project-setup-action/domains/cypress_tests.txt

```diff
@@ -0,0 +1 @@
+0.0.0.0     test.zango.local
```

Comment:
Should this file be placed inside ``actions/zango-project-setup-action/domains/`` , or not? This folder is intended for general Zango project setup actions. It might be more appropriate to place this file in the directory where the specific workflow that uses this action.

Let me know your thoughts.

Reply:
make sense. I'll move it to `workflows/cypress/`.